movies.html

```diff
@@ -0,0 +1,134 @@
+        <div class=" row seccion-uno col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div class=" row col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                <div class="seccion-dos">
+                    <div class="seccion1movies">
+                        <h1>Películas en tendencia</h1> 
+                    </div>
+                </div>
+            </div> 
+                    
+                
+            <div class="row col-xs-6 col-sm-6 col-md-6 col-lg-6">  
+                <div class="seccion-dos">
+                    <div class="seccion2movies">
+                        <a class="listapeliculas" href="movies-list.html">Ver lista<img  class="lista" src="imagenes/iconolista.png.png"></a> 
+                    </div> 
+                </div>           
+            </div>
+        </div>  
+    </div> 
+
+<div class="container">
+    <div class="row">
+        <div class="seccion-dos col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div id="seccion3movies">
+                <section class="seccion3categorias-etiquetas">
+                <aside class="col-xs-4 col-sm-4 col-md-4 col-lg-4">
+                    <div class="box">
+                        <div class="container-1">
+                            <span class="icon"><i class="fa fa-search"></i></span>
+                            <input type="search" id="search" placeholder="Search..." />
+                            <br/><br/>
+                        </div>
+                    </div>
+                    <h2 class="titulo">Categorías</h2>
+                    <hr/>
+                    <input type="checkbox" id="acción" name="peliculas" value="accion"/>
+                    <label for="acción">Acción</label><br/><br/>
+                    <input type="checkbox" id="comedia" name="peliculas" value="comedia"/>
+                    <label for="comedia">Comedia</label><br/><br/>
+                    <input type="checkbox" id="drama" name="peliculas" value="drama"/>
+                    <label for="drama">Drama</label><br/><br/>
+                    <input type="checkbox" id="fantasia" name="peliculas" value="fantasia"/>
+                    <label for="fantasia">Fantasia</label><br/><br/>
+                    <input type="checkbox" id="horror" name="peliculas" value="horror"/>
+                    <label for="horror">Horror</label><br/><br/>
+                    <br/><br/>
+                    <h2 class="titulo">Etiquetas</h2>
+                    <hr/>
+                    <a href="" class="botonetiquetas">Diversión</a>
+                    <a href="" class="botonetiquetas">Misterio</a>
+                    <a href="" class="botonetiquetas">Terror</a>
+                    <a href="" class="botonetiquetas">Acción</a>
+                    <a href="" class="botonetiquetas">Ficción</a>
+                    
+                </aside> 
+            </section>
+            <div class="col-xs-8 col-sm-8 col-md-8 col-lg-8">
+                <section class="seccion3">

```

Comment:
Notice that the `seccion3` elements have the property `float: left;` this causes the container element to collapse, if there were a sibling element this might cause a wrong displacement of sibling elements. It is recommended to clear float properties. Reference [CSS Clear: Learn to Clear Floating Elements](https://www.bitdegree.org/learn/css-clear#:~:text=Clearing%20floats,is%20not%20supposed%20to%20float.)